docs/apache-airflow-providers-postgres/commits.rst

```diff
@@ -36,6 +48,7 @@ Latest change: 2022-09-05
 =================================================================================================  ===========  ===========================================================================
 Commit                                                                                             Committed    Subject
 =================================================================================================  ===========  ===========================================================================
+`25d0baa4ee <https://github.com/apache/airflow/commit/25d0baa4ee69769ff339931f76ebace28c4315f2>`_  2022-09-05   ``Prepare bug-fix release of providers out of band (#26109)``
 `54e7d20176 <https://github.com/apache/airflow/commit/54e7d20176a08ecee659cdd9844c75f4f7570a11>`_  2022-09-05   ``Bump dep on common-sql to fix issue with SQLTableCheckOperator (#26143)``
 `14d56a5a9e <https://github.com/apache/airflow/commit/14d56a5a9e78580c53cf85db504464daccffe21c>`_  2022-08-23   ``postgres provider: use non-binary psycopg2 (#25710)``
 `ca9229b6fe <https://github.com/apache/airflow/commit/ca9229b6fe7eda198c7ce32da13afb97ab9f3e28>`_  2022-08-18   ``Add common-sql lower bound for common-sql (#25789)``
```

Comment:
in the change log this entry was moved from 5.2.1 yo 5.2.2 so there is inconstancy

Reply:
Yep.